.gitmodules

```diff
@@ -5,3 +5,6 @@
 	path = submodules/openssl
 	url = https://github.com/quictls/openssl.git
 	branch = OpenSSL_1_1_1k+quic
+[submodule "submodules/clog"]
+	path = submodules/clog
+	url = https://github.com/microsoft/CLOG.git
```

Comment:
Should this be pointing to a particular `branch` like we do for openssl above or is `master` what's being used here?

Reply:
If we can merge 3.0 into CLOG master first, then this should be good as is. That'd be my preference.